faqs/galaxy/datasets_sorting_reference_genome.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Sorting Reference Genome
+area: datasets  
```

Comment:
there are a few FAQs about reference genomes, maybe they deserve their own `area`?

Reply:
> there are a few FAQs about reference genomes, maybe they deserve their own `area`?

Hi @beatrizserrano I changed the area to "reference genome".Do you suggest me to change the  areas of the other PRs?